lib/compress/zstd_compress.c

```diff
@@ -1932,6 +1932,17 @@ ZSTD_reset_matchState(ZSTD_matchState_t* ms,
         ZSTD_cwksp_clean_tables(ws);
     }
 
+    if (ZSTD_rowMatchFinderUsed(cParams->strategy, useRowMatchFinder)) {
+        /* Row match finder needs an additional table of hashes ("tags") */
+        size_t const tagTableSize = hSize * sizeof(U16);
```

Comment:
Investigate if we really need `U16` space for the tags table.

Reply:
For now we need it, but will look into reducing footprint soon.